RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -142,20 +142,51 @@ struct SubscriptionDetailsView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Text("\(subscriptionInformation.title) - \(subscriptionInformation.durationTitle)")
-                .font(.subheadline)
-                .padding([.horizontal, .top])
-
-            Text("\(subscriptionInformation.price)")
-                .font(.caption)
-                .foregroundColor(Color.gray)
-                .padding(.horizontal)
-
-            if let nextRenewal =  subscriptionInformation.nextRenewalString {
-                Text("\(subscriptionInformation.renewalString): \(String(describing: nextRenewal))")
+            VStack(alignment: .leading) {
+                Text("\(subscriptionInformation.title)")
+                    .font(.headline)
+                Text("\(subscriptionInformation.explanation)")
+                    .frame(maxWidth: 200, alignment: .leading)
                     .font(.caption)
                     .foregroundColor(Color.gray)
-                    .padding([.horizontal, .bottom])
+            }.padding([.bottom], 10)
+
+            HStack(alignment: .center) {
+                Image(systemName: "coloncurrencysign.arrow.circlepath")
+                    .frame(width: 22)
```

Comment:
Since this is repeated, I would extract to a constant that can be reused.

Reply:
Good one! Done: https://github.com/RevenueCat/purchases-ios/pull/4072/commits/65ecb16c3cf097ab304d031c6eda0febf0d1a9a2